examples/ecr/ecr-full-repo.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: "ecr.services.k8s.aws/v1alpha1"
+kind: Repository
+metadata:
+    name: "test-full-repository-from-ack"
+spec:
+    repositoryName: "test-full-repository-from-ack"
+    encryptionConfiguration:
+        encryptionType: KMS
+        # This key needs to exist already
+        kmsKey: "9d319c68-60ea-4ed6-8a33-2e09086fad2d"
+    imageScanningConfiguration:
+        scanOnPush: true
+    imageTagMutability: true
+    tags:
+    - tag1:
+        key: "is-encrypted"
+        value: "true"
+    - tag2:
+        key: "is-scanned"
+        value: "true"
```

Comment:
Ah, no, this is slightly off. For ECR, the Tags field is `[]*Tag`, where `Tag` is a struct with two string fields, Key and Value:
```json
    "Tag":{
      "type":"structure",
      "members":{
        "Key":{"shape":"TagKey"},
        "Value":{"shape":"TagValue"}
      }
    },
    "TagKey":{"type":"string"},
    "TagKeyList":{
      "type":"list",
      "member":{"shape":"TagKey"}
    },
    "TagList":{
      "type":"list",
      "member":{"shape":"Tag"}
    },
```

https://github.com/aws/aws-sdk-go/blob/20cd465df9cd1c9f5c941d7843d29704b9510751/models/apis/ecr/2015-09-21/api-2.json#L1570-L1585

So, the above should be this instead:

```yaml
    tags:
    - key: "is-encrypted"
      value: "true"
    - key: "is-scanned"
      value: "true"